_pages/getting-started/getting-started.md

```diff
@@ -123,7 +123,7 @@ Maybe you've identified a good way to update the script!
 If you are running a project locally, you may also find yourself needing to know
 [some basics of the Mac OS X command line](https://blog.teamtreehouse.com/introduction-to-the-mac-os-x-command-line).
 
-And remember, if you can't figure out how to get it running, that's probably not your fault --- poor documentation is a defect.
+And remember, if you can't figure out how to get it running, that's probably not your fault --- there is likely something that needs to be updated, and that can be accomplished by fillint out the [TTS Tech Portfolio General Request Form](https://forms.gle/wPrCXJqDgaksXRwV8) and selecting the option for a _Generic Request_.
```

Comment:
I don't think that this is necessarily a good spot to direct them to the general request form. The docs are about command line basics and the laptop script. And in particular, reassuring folks that poor documentation isn't a fault of theirs. I'm not sure I see the connection to the general request form?

Reply:
Gargh, just noticed a typo too. 

Anyways -- I see what you are saying now. I see this section more about the mac laptop from an equipment management perspective rather than a coding/documentation best practices. I agree that "poor documentation is a defect" I just don't think this is the best place where it should be mentioned. I think it is better suited in a How We Work, Writing Guide or Engineering Guide. I was coming at it from the angle that the laptop script should already be properly documented & updated. The connection with the General Request Form is that the TTS Tech Portfolio is the maintainers of the laptop script [I think, @afeld & @JJediny can help verify]. So, we should be keeping it up to date and with the proper documentation, and one of the ways we're collecting tasks like this is through the General Request form.

Reply:
Well, i see your point too! With y'all actively maintaining it now, this section could probably be trimmed way back. How bout something like "If you have trouble with the laptop script, [ping the Infrastructure team for help](link to form)."